Lib/test/test__xxsubinterpreters.py

```diff
@@ -195,6 +196,29 @@ def test_bool(self):
     def test_float(self):
         self._assert_values([0.0, 1.1, -1.0, 0.12345678, -0.12345678])
 
+    def test_tuple(self):
+        self._assert_values([(), (1,), ("hello", "world", ), (1, True, "hello")])
+        # Test nesting
+        self._assert_values([((1,),), ((1, 2), (3, 4)), ((1, 2), (3, 4), (5, 6))])
```

Comment:
Hmm, that's a tad hard to read.  Consider putting one entry per line:

```suggestion
        self._assert_values([
            ((1,),),
            ((1, 2), (3, 4)),
            ((1, 2), (3, 4), (5, 6)),
        ])
```